### PR TITLE
Converting Finderchart to JoyUI

### PR DESCRIPTION
**Pull request by kpuriIpac**
**[Firefly-1417](https://jira.ipac.caltech.edu/browse/FIREFLY-1417)**: 
- irsa-ife PR: https://github.com/IPAC-SW/irsa-ife/pull/308 
- The real changes here are only lines 24-25. Added a `Stack` around the label/checkbox so that they appear on the same line next to each other (as opposed to label on one line and checkbox on another)
- Also the label now has an optional `minWidth` property (`labelMinWidth`, default value 0). This is useful when there are multiple labels and you'd want the checkboxes' start positions to be the same. (Click on `Image Search Options` on finderchart to see this in action).  

**Updates 3/12**: 
- small change in SearchPanel thanks to @loitly! but fixes Finderchart list of targets/multiple positions search. 
- test build is updated below, refer to ife PR above for other cleanup & changes. 

**Testing**:  
   - Finderchart: https://firefly-1417-finderchart-conversion.irsakudev.ipac.caltech.edu/applications/finderchart

### CODE REVIEW

**Comment by lrebull** [comment]
(random selfish promotion of my own ulterior motives: somehow, Finder Chart didn't pick up a fix from long ago re: "new" catalog behavior. Could you make sure that the Joy UI build actually does pick up that fix? https://jira.ipac.caltech.edu/browse/IRSA-4046)

File: src/firefly/js/ui/CheckboxGroupInputField.jsx [hunk 1]
```diff
@@ -17,33 +17,34 @@ function convertValue(value,options) {
 }
 
 export function CheckboxGroupInputFieldView({fieldKey, onChange, label, tooltip:toggleBoxTip, slotProps, type,
-                                             options, alignment:orientation, value:fieldValue, sx}) {
-
+                                             options, alignment:orientation, value:fieldValue, sx, labelMinWidth=0}) {
```
Comment (loitly) [inline src/firefly/js/ui/CheckboxGroupInputField.jsx:20]: This is not needed.  To set minWidth on label, use `slotProps.label`.

File: src/firefly/js/ui/CheckboxGroupInputField.jsx [hunk 2]
```diff
@@ -18,32 +18,33 @@ function convertValue(value,options) {
...
-                            <FormControl key={value}>{
-                                //until https://github.com/mui/material-ui/issues/37764 & related issues are fixed,
-                                //we need to wrap `cb` in a FormControl as a workaround: https://stackoverflow.com/a/66738444/8252556
-                                tooltip
-                                    ? <Tooltip {...{title: toggleBoxTip, key: value}}> {cb} </Tooltip>
-                                    : cb
-                            }</FormControl>
-                        );
-                    })}
+                <Stack direction='row' alignItems='center' spacing={2}>
```
Comment (loitly) [inline src/firefly/js/ui/CheckboxGroupInputField.jsx:24]: This is most likely not right.  It's a reusable component and cannot be styled like this.  I think direction is handled by FormControl orientation.

Comment (kpuriIpac) [reply src/firefly/js/ui/CheckboxGroupInputField.jsx]: I had totally missed this. Got rid of it now! Just setting `orientation` (alignment) directly from where `CheckboxGroupInputField` is called in FinderChart.

**Comment by kpuriIpac** [comment]
@robyww @loitly can one of you approve this if no pending issues? ready to merge this and the ife PR then.

**Review by robyww — APPROVED** [review]

**Pull request merged**